plugins/volumeFilter_test.go

```diff
@@ -1111,7 +1139,12 @@ func runTestVolumeFilterFn(
 			mode:                     mode,
 		}
 
-		actual, e := volumeFilterFn(dailyOTB, dailyTBBAccumulator, inputOp, utils.NativeAsset, utils.NativeAsset, lp)
+		// For our tests, the selling asset is QUOTE, and the buying asset is XLM
+		// To identify a sell op, we check if the base is the selling asset and the quote is buying
```

Comment:
This part confuses me a little.

1. The `volumeFilterFn` isn't aware of whether we are sell mode or buy mode but is more concerned with base and quote asset, so naming the parameters to the function as baseAsset and quoteAsset would be a little less confusing.
1. I was expecting the base asset to always be the native asset and the quote asset to be the quote asset. I think these are passed in reverse over here, but what makes this particularly confusing is that the test is passing this way. I think the comments below will help fix this

Reply:
Yup, the comments below fixed this. Makes sense, since it just switches the buying and assets in the operations - which means that switching the parameters here also ensures correctness.